src/pages/Login.vue

```diff
@@ -23,16 +23,13 @@
           .text-caption.text-white You do not have accounts on polkadot js extension
         .text-caption.text-center.text-white.q-mt-lg OR
         #googleLogin.q-mt-lg
-            q-btn.full-width.btnGoogle(
-              no-caps
-              label="Login with google"
-              disabled
-            )
+          #google-signin-btn.flex.justify-center(class="g-signin2")
 </template>
 
 <script>
 import { AccountsMenu, SelectedAccountBtn } from '~/components/common/index.js'
 import { mapGetters } from 'vuex'
+import Jwt from '../../utils/Jwt'
```

Comment:
please use ~ to refer to "/src/" dir, this evit issues with storybook

Reply:
It's necessary to decode the credentials and get the account information because the response is only a token